utils/build-builder.mjs

```diff
@@ -11,7 +11,6 @@ import { esbuild } from './build.mjs';
 
 const pkgPath = join(process.cwd(), 'package.json');
 const pkg = JSON.parse(readFileSync(pkgPath, 'utf8'));
-
```

Comment:
Were you planning to extend this script to support more options?

Reply:
Not so far. This extra newline was just bugging me.